graphics/zimg/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        sekrit-twc zimg 2.9.3 release-
+
```

Comment:
Please add the `revision 0` line here. We're getting into the habit of always adding the revision line, even when it's 0, to make it easier to see where a revision in a port should be increased, when that's needed.

Reply:
Added.  If `revision` is becoming policy, then it sounds like `port lint` should warn when ports are missing it?